source/includes/kubernetes/_k8_configmaps.md

```diff
@@ -129,6 +129,61 @@ Return value:
 | `taskId` <br/>_string_     | The id corresponding to the create config maps task. |
 | `taskStatus` <br/>_string_ | The status of the operation.                         |
 
+<!-------------------- REPLACE A CONFIG MAP -------------------->
+
+#### Replace a config map
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/configmaps/configmaps-name/default"
+  Content-Type: application/json
+  {
+  "apiVersion": "v1",
+  "kind": "ConfigMap",
+  "metadata": {
+    "name": "game-demo"
+  },
+  "data": {
+    "player_initial_lives": "5",
+    "ui_properties_file_name": "user-interface.properties",
+    "game.properties": "enemy.types=aliens,monsters\nplayer.maximum-lives=5\n",
+    "user-interface.properties": "color.good=purple\ncolor.bad=yellow\nallow.textmode=true\n"
+  }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/configmaps/:id</code>
+
+Replace a config map in a given [environment](#administration-environments).
+
+| Required Attributes           | &nbsp;                                                                   |
+| ----------------------------- | ------------------------------------------------------------------------ |
+| `apiVersion` <br/> _string_   | The api version (versioned schema) of the config map.                    |
+| `metadata` <br/>_object_      | The metadata of the config map.                                          |
+| `metadata.name` <br/>_string_ | The name of the config map.                                              |
+| `data`<br/>_object_           | The non-confidential data (in key-value pairs) stored in the config map. |
+
+| Optional Attributes                | &nbsp;                                           |
+| ---------------------------------- | ------------------------------------------------ |
+| `metadata.namespace` <br/>_string_ | The namespace in which the config map is created |
+
+Return value:
+
+| Attributes                 | &nbsp;                                               |
+| -------------------------- | ---------------------------------------------------- |
+| `taskId` <br/>_string_     | The id corresponding to the create config maps task. |
```

Comment:
`The id corresponding to the ~create~ replace config maps task.`

Reply:
😳 29bd284da5bdcce53262efad30eeba8c620e8c97